src/bls/bls.h

```diff
@@ -42,7 +42,7 @@ class CBLSWrapper
     friend class CBLSPublicKey;
     friend class CBLSSignature;
 
-    bool fLegacy;
+    bool fLegacy{false};
```

Comment:
I don't see a code path that leaves `fLegacy` uninitialized. And if anything it should be assigned to `fLegacyDefault`, not to false